src/lib/types/user.ts

```diff
@@ -33,7 +33,7 @@ export interface IUser {
     scimId?: string;
 }
 
-export type MinimalUser = Pick<
+export type MinimalUiUser = Pick<
```

Comment:
What if this data is accessed through the API? Would it still make sense to call it "Ui"

Reply:
good point. The data is intended to be used to render a user (so it includes the image url, notably). I don't mind leaving or removing that part of the name. Or if you have a different suggestion, I'm very open to that too.

Reply:
Personally I'd stick to the previous name

Reply:
Done ✅